### PR TITLE
Remove references to the timing bit.

### LINKED ISSUE #827: Cleanup of timing bit
When the timing bit was removed from mcontrol6, not all references were cleaned up.  For instance:
- The dcsr section says "Note that mcontrol/mcontrol6 triggers with timing=after are considered to be high priority causes on the subsequent instruction."  That should probably be something like "mcontrol triggers with timing=after and mcontrol6 triggers which are firing after the instruction which hit the trigger" (or maybe "and mcontrol6 triggers which will write hit1=1 upon firing").
- dpc holds "The address of the next instruction to be executed at the time that debug mode was entered. If the trigger is mcontrol or mcontrol6 and timing is 0, this corresponds to the address of the instruction which
caused the trigger to fire."  That should say something similar to above where mcontrol and mcontrol6 are treated slightly differently.
- B.2.9 has several examples with timing=1.  I also notice that tdata1 should contain the correct type field in each of the examples.

### PR DESCRIPTION
**Pull request by pdonahue-ventana**
Fixes #827.  Also changed trigger examples to fix some bugs, add the type field, add vs/vu in some examples, enhance one example to include textra, etc.

### CODE REVIEW

**Review by timsifive — CHANGES_REQUESTED** [review]

File: debugger_implementation.tex [hunk 1]
```diff
@@ -451,69 +451,75 @@ \subsection{Triggers}
...
 \medskip
 
-\noindent Enter Debug Mode right after the value at 0x80007f80 is read:
+\noindent Enter Debug Mode when performing a load at address 0x80007f80 in
+M-mode or S-mode or U-mode:
 
 \begin{tabulary}{\textwidth}{|r|r|L|}
     \hline
-    \RcsrTdataOne & 0x4159 & timing=1, action=1, match=0, m=1, s=1, u=1, load=1 \\
+    \RcsrTdataOne & 0x68005159 & type=6, dmode=1, action=1, select=0, match=0, m=1, s=1, u=1, load=1 \\
```
[the remark this answers is not in the document]
Comment (pdonahue-ventana) [reply debugger_implementation.tex:473]: Oh yeah.  I noticed that there was a bug in the original spec value but then I only partially fixed it.

File: debugger_implementation.tex [hunk 2]
```diff
@@ -451,69 +451,75 @@ \subsection{Triggers}
...
     \RcsrTdataTwo 0 & 0x80007c80 & start address (inclusive) \\
     \hline
-    \RcsrTdataOne 1 & 0x11da & action=1, match=3, m=1, s=1, u=1, store=1 \\
+    \RcsrTdataTwo 0 & 0x03000000 & mhselect=6, mhvalue=0 \\
+    \hline
+    \RcsrTdataOne 1 & 0x69801182 & type=6, dmode=1, action=1, select=0, match=3, vs=1, vu=1, store=1 \\
     \hline
     \RcsrTdataTwo 1 & 0x80007cf0 & end address (exclusive) \\
     \hline
+    \RcsrTdataTwo 1 & 0x03000000 & mhselect=6, mhvalue=0 \\
```
[the remark this answers is not in the document]
Comment (timsifive) [reply debugger_implementation.tex:496]: I don't see this addressed.

File: debugger_implementation.tex [hunk 3]
```diff
@@ -451,69 +451,75 @@ \subsection{Triggers}
...
 
 \begin{tabulary}{\textwidth}{|r|r|L|}
     \hline
-    \RcsrTdataOne 0 & 0x195a & action=1, chain=1, match=2, m=1, s=1, u=1, store=1 \\
+    \RcsrTdataOne 0 & 0x69801902 & type=6, dmode=1, action=1, chain=1, select=0, match=2, vs=1, vu=1, store=1 \\
     \hline
     \RcsrTdataTwo 0 & 0x80007c80 & start address (inclusive) \\
     \hline
-    \RcsrTdataOne 1 & 0x11da & action=1, match=3, m=1, s=1, u=1, store=1 \\
+    \RcsrTdataTwo 0 & 0x03000000 & mhselect=6, mhvalue=0 \\
```
[the remark this answers is not in the document]
Comment (timsifive) [reply debugger_implementation.tex:490]: This one either.

**Review by timsifive — CHANGES_REQUESTED** [review]

**Review by pdonahue-ventana — COMMENTED** [review]
I thought that I added all the changes to a batch and then committed that batch.  I guess not.  Unless I screwed something up again, they should all be there now.

**Review by timsifive — APPROVED** [review]

**Pull request merged**